app/resources/organiser_resource.py

```diff
@@ -45,4 +46,49 @@ def insert_data(self, organiser: Organiser):
 
         return result
 
+    def modify_data(self, organiser: Organiser):
 
+        d_service = self.data_service
```

Comment:
Can you please move this logic to [MySqlRdbDataService.py](https://github.com/SaaSy-Pants/SaaSy-Events-usermgmt/blob/main/framework/services/data_access/MySqlRdbDataService.py)?

Refer other methods in this class and in the service class.